src/app/code/Komoju/Payments/Api/KomojuApi.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+/** Replace functionality with KomojuApi package once available **/
```

Comment:
@thoiberg A quick search suggests we should be able to use 3rd party dependencies so I've added this quick comment in as you suggested 👍🏻